templates/bin/fastcheck

```diff
@@ -5,7 +5,7 @@ set -e
 if ! bundle exec rubocop -D -c .rubocop.yml --fail-fast
 then
   echo 'rubocop detected issues!'
-  bundle exec rubocop -a -D -c .rubocop.yml
+  bundle exec rubocop -A -D -c .rubocop.yml
   echo 'Tried to auto correct the issues, but must be reviewed manually, commit aborted'
```

Comment:
Doesn't this echo in either case?

Reply:
Yes. I want to add the message "review manually"